Ginger/GingerTest/POMs/Common/ucGridPOM.cs

```diff
@@ -74,17 +74,30 @@ public void ClickOnCheckBox(string checkboxHeaderValue, string fieldToSearchOnHe
                             checkbox = General.GetVisualChild<CheckBox>(cell);
                         }
 
-                        if (checkbox.IsChecked == true)
+                        if (!prioritizeLocators || (prioritizeLocators && clickCheckBox))
```

Comment:
ucGridPOM is generic class to handle all ucGrid's we have and not specificaly for POM needs (name can be confusing).
please do not use specific names like "prioritizeLocators" for parameters it needs to be generic handling.
create seperate function which will do ucGrid rows re-order in generic way, "ClickOnCheckBox" method should be used only for checkbox's selection

Reply:
Done. Used generic names for the Parameter in ucGridPOM file. Also, handled reordering in generic way.